test/forms/helpers/read-files-as-data-urls.test.js

```diff
@@ -0,0 +1,35 @@
+import { readFilesAsDataUrls } from '../../../src'
+
+describe('readFilesAsDataUrls', () => {
```

Comment:
I believe the switch from `node` to `jsdom` is what flipped the 66% to 33%. We did however change the implementation so that `readFiles` should only ever be called after user input (on change). While the `isServer()` guard is probably no longer needed, I'd like to keep this a minor change by writing the test for the functionality as written today, instead of removing that guard.

It's definitely a confusing topic so let me know if you have any questions!